From f1ba779e2b64ca5168146811f4893b222ef1dee9 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 10 Jun 2024 10:52:05 +0200
Subject: [PATCH 0942/1695] plat-stm32mp2: enable GIC PM support

Enable CFG_ARM_GIC_PM for plat-stm32mp2 for restoration of OP-TEE
interrupts configuration in PM resume sequence.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git b2e78442fece24f3bb99767a1b12c6b913e9bbdf plat-stm32mp2: enable GIC PM support]
Change-Id: I37bf2e1acb949ec4487d25a0ecd70007871be57e
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/386309
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 6846b4840..09b715c61 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -79,6 +79,7 @@ endif
 include core/arch/arm/cpu/cortex-armv8-0.mk
 supported-ta-targets ?= ta_arm64
 
+$(call force,CFG_ARM_GIC_PM,y)
 $(call force,CFG_ARM64_core,y)
 $(call force,CFG_CORE_ASYNC_NOTIF,y)
 $(call force,CFG_CORE_ASYNC_NOTIF_GIC_INTID,31)
-- 
2.47.2

